www/src/css/index.css

```diff
@@ -112,34 +88,17 @@ a:hover,
   display: none;
 }
 
-.navbar {
-  box-shadow: none;
-}
-
-.navbar__title {
-  font-size: 1.2rem;
-  margin-left: 0.2rem;
-  display: none;
-}
-
-.navbar-sidebar__brand .navbar__title {
-  display: inline;
+hr {
+  margin-top: 2.5rem;
+  border-color: #ddd;
 }
 
-@media screen and (min-width: 420px) {
-  .navbar__title {
-    display: inline;
-  }
+html[data-theme="dark"] hr {
+  border-color: #242526;
 }
 
-.home-subtitle {
-  justify-content: center;
-  padding: 0rem 0 2rem 0;
-  opacity: 0.6;
-  font-style: italic;
-  margin: 0 auto;
-  font-size: 1.2rem;
-  text-align: center;
+html[data-theme="dark"] {
+  --ifm-background-color: #1f201c;
 }
```

Comment:
I would move variable declarations and their overrides at the top of this `index.css` file, so they're easy to discover 👍🏽

Reply:
Good call :+1: